user_list.php

```diff
@@ -29,64 +29,46 @@
 			<!-- import the webpage's javascript -->
 			<!-- nav javascript -->
 			<script src="js/nav.js" defer></script>
-			<!-- back to top button javascript -->
-			<script src="js/backtotop.js" defer></script>
```

Comment:
#I trialled not having a back-to-top button in my second layout option and instead having a sticky navigation menu